applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py

```diff
@@ -50,6 +50,33 @@ def __init__(self, model, project_parameters):
 
         super(StructuralMechanicsAnalysis, self).__init__(model, project_parameters)
 
+    def Initialize(self):
+        """ Initializing the Analysis """
+        super(StructuralMechanicsAnalysis, self).Initialize()
+        self._GetSolver().SetEchoLevel(self.echo_level)
```

Comment:
Why?
The solver has its own echo level
I propose to do this only if there is no echo level given in the solver settings

Reply:
This was the reason I have done this in two steps, first remove the solver wrappers and later unify the analysis. I have kind of conflictive changes in the ContactStructuralMechanicsApplication, and it is basically because we print many things that usually conflict what I actually want to see. I want to see a proper table like this:

![imagen](https://user-images.githubusercontent.com/19991680/66255635-4291dc00-e786-11e9-9347-8bd356e5c640.png)

Not a discontinuous thing that no one can understand what is happening